prac_06/guitars.py

```diff
@@ -0,0 +1,33 @@
+"""CP1404/CP5632 Practical - Guitars"""
+from prac_06.guitar import Guitar
+
+
+def main():
+    print("My guitars!")
+    guitars = get_guitars()
+    print("These are my guitars:")
+    print_guitars(guitars)
+
+
+def print_guitars(guitars):
+    """Print information about stored guitars"""
+    for i, guitar in enumerate(guitars, 1):
+        vintage_string = "(Vintage)" if guitar.is_vintage() else ""
+        print(
+            f"Guitar {i}: {guitar.name:{max(len(guitar.name) for guitar in guitars)}} ({guitar.year}), worth ${guitar.cost:.2f} {vintage_string}")
+
+
+def get_guitars():
```

Comment:
Nice use of functions. `main` is easy to read.

Reply:
Thank you for the feedback.